### PR TITLE
23.11-tapir: add artwork

### PR DESCRIPTION
**Pull request by the-furry-hubofeverything**
tapir
Artworks under CC BY-SA 4.0, based on @riotbib's work on #105, for #104

Stock image "Malayan Tapir" by Tucky Piyapong from Pexels, under "Pexels License", which allows modifications and is free to use.

Inkscape project file is included as `tapir.svg`.

Feedback is welcome. There was some concerns about the license, so let me know if this needs to be relicensed.

### CODE REVIEW

**Comment by rjpcasalino** [comment]
Looks great! Wonderful work

**Comment by RaitoBezarius** [comment]
@the-furry-hubofeverything Hi there, I think it's a great logo, though I confirm that @riotbib interpretation is right that if we had to print those stickers, we would need to perform attribution on the print, for the NixOS logo, it should be the same honestly, the reason for why no one is doing it is I assume that no one expect the Foundation to chase you with the CC BY attribution clause for the NixOS logo. In this instance, if we went with your logo, it's not easy to have the same expectations.

As to not set a bad precedent, I'd appreciate it if we could have a CC-SA for example but this doesn't exist as far as I know.
If there was an exception for "physically" printed medias, that'd be awesome, as for virtual medias, it makes sense to perform the CC BY attribution.

**Comment by RaitoBezarius** [comment]
Also, I am aware of https://wiki.creativecommons.org/wiki/Best_practices_for_attribution#Publishing_a_web_page_for_attribution but not sure if this is legal advice per se.

**Comment by RaitoBezarius** [comment]
Thanks to @mclegrand, apparently, the license text says:

> 2. You may satisfy the conditions in Section 3(a)(1) in any reasonable manner based on the medium, means, and context in which You Share the Licensed Material. For example, it may be reasonable to satisfy the conditions by providing a URI or hyperlink to a resource that includes the required information.

Therefore, it **might** be sufficient to design stickers and provide at sharing time (e.g. when distributing stickers) a way to observe/see the attribution, via a link or the full attribution data.

**Comment by the-furry-hubofeverything** [comment]
> the reason for why no one is doing it is I assume that no one expect the Foundation to chase you with the CC BY attribution clause for the NixOS logo.

I don't understand, can you rephrase?

Also, the [shirt designs](https://github.com/NixOS/nixos-artwork/blob/master/CONTRIBUTORS.md#betalars-nea-insomnia-nek0) are also created with CC BY-SA 4.0, how was that distributed?

Also, in the end, if there's any problems, I think I can add additional permissions specifically for parties that require them.

**Comment by the-furry-hubofeverything** [comment]
> Thanks to @mclegrand, apparently, the license text says:
> 
> > 2. You may satisfy the conditions in Section 3(a)(1) in any reasonable manner based on the medium, means, and context in which You Share the Licensed Material. For example, it may be reasonable to satisfy the conditions by providing a URI or hyperlink to a resource that includes the required information.
> 
> Therefore, it **might** be sufficient to design stickers and provide at sharing time (e.g. when distributing stickers) a way to observe/see the attribution, via a link or the full attribution data.

My idea with this license is that it's not restrictive like with "All rights reserved", but I'm uncomfortable with going public domain. I don't intend to make it difficult to distribute, but this kind of licensing is what I'm used to.

For different mediums like 3d printing, there's an established precedent with tags (e.g. [thingiverse attribution cards](https://www.thingiverse.com/thing:6293669/attribution_card)). 

I think under the spirit of the license, the idea of reasonable attribution is that people would at least be able to see it without having trouble finding it.

**Comment by the-furry-hubofeverything** [comment]
> I think under the spirit of the license, the idea of reasonable attribution is that people would at least be able to see it without having trouble finding it. 

Like, a slip of paper on the table when distributing, or the packaging with a piece of paper included... etc

**Comment by samueldr** [comment]
Hm, what's up with the lower left lambdaflake part?

It's not the blue it should be, and it is (partly) covered by the lighter blue one.

**Comment by the-furry-hubofeverything** [comment]
> I used nix-snowflake.svg, Inkscape for the layout and clipping, and krita for the level adjustment

Let me double check

**Comment by the-furry-hubofeverything** [comment]
So the light color from the bottom left corner comes from the `gradient-logo` group layer. `print-logo` has the darker blue lambda

**Comment by the-furry-hubofeverything** [comment]
How's this? @samueldr 

tapir

**Comment by RaitoBezarius** [comment]
> > I think under the spirit of the license, the idea of reasonable attribution is that people would at least be able to see it without having trouble finding it.
> 
> Like, a slip of paper on the table when distributing, or the packaging with a piece of paper included... etc

I agree with you, if that's fine with you, we will go with that.

**Comment by riotbib** [comment]
Just briefly commenting, it's a tiny bit weird having a license put on the version of the Tapir – cropped by me. But do feel free to go on, since I do not care about intellectual property.

**Comment by riotbib** [comment]
Also: My concerns were about the original picture. You'd need to oblige to the [Pexel's license](https://www.pexels.com/license/), since this is the source of the picture. Which should be fine; but I am not a lawyer.

**Comment by the-furry-hubofeverything** [comment]
> Also: My concerns were about the original picture. You'd need to oblige to the [Pexel's license](https://www.pexels.com/license/), since this is the source of the picture. Which should be fine; but I am not a lawyer.
 
I forgot to check!

Briefly skimming the [terms of service on pexel](https://www.pexels.com/terms-of-service/):
> 6.2. Furthermore, despite the Pexels License provided to you by Pexels hereunder you agree to NOT use any Pexels Content made available via the Service (in whole or in part):
[...]
> - as a trademark, service mark, logo or other indication of origin, or as part thereof;

Oh dear.

So, Pexel license might not work for this - which sucks, since I bet @riotbib have put some work into this... I know I have. 

It might not apply to derivatives, but I'm not a lawyer. I don't know if there's any examples of the use of adapted Pexel content in logos.

Creative commons is widespread, and quite open, so I'm pretty sure that finding an logo using adapted content licensed with CC is much simpler than finding a logo using adapted content licensed under Pexel's license, meaning, in my opinion, it might be safer to use CC stocks rather than Pexel stocks. Again, I'm not a lawyer.

lmk if this is an issue. I'm going to stop making updates to this PR, and only continue if the risk is acceptable. I'm going to attempt to look for a better source

cc @figsoda @RaitoBezarius @mclegrand

**Comment by rjpcasalino** [comment]
how about using one in the public domain? @the-furry-hubofeverything thanks for your hard work with this! https://commons.wikimedia.org/wiki/File:Tapirus_terrestris_laying_down.jpg

**Comment by the-furry-hubofeverything** [comment]
Closing this PR in favour of #107, which includes an alternative stock under CC0. 

If this is still preferred I can reopen it
